src/calculus.jl

```diff
@@ -252,10 +252,9 @@ evaluated at the vector `vals`. If `vals` is omitted, it is evaluated at zero.
 """
 function jacobian(vf::Array{TaylorN{T},1}) where {T<:Number}
     numVars = get_numvars()
-    @assert length(vf) == numVars
-    jac = Array{T}(undef, numVars, numVars)
+    jac = Array{T}(undef, numVars, length(vf))
 
-    @inbounds for comp = 1:numVars
+    @inbounds for comp = 1:length(vf)
```

Comment:
I think the following is recommended (it should have been changed before, but ...):
```suggestion
    @inbounds for comp in eachindex(vf)
```